### PR TITLE
Card gallery

### LINKED ISSUE #5: [feat] Instructor card collection
User Story: 
As a User, I want to see different Instructors, to choose from.

AC:
- [x] Card Information is stored in zustand
- [x] Cards are created from Information
- [x] Collection displays in Chip-layout
- [x] Collection can change between layouts

Complexity Index: S | S

### PR DESCRIPTION
**Pull request by haskamp**
## Motivation
- Cards are created from form information out of storage
 - Card gallery displays also in chip-layout
 - Toggle button can change between layouts

## Issues closed
#5 
Screenshot 2022-02-06 at 20 25 52

### CODE REVIEW

File: src/molecules/card-chip-main/index.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+				<CardActions>
+					<Tooltip title="Expand">
+						<IconButton
+							aria-expanded={cards[index].isExpanded}
+							aria-label="show more"
+							onClick={() => {
+								setExpanded(index);
+							}}
+						>
+							<ExpandMore />
```
Comment (pixelass) [inline src/molecules/card-chip-main/index.jsx:48]: You could rotate the icon or use a different icon when expanded. 

Right now, the Icon does not indicate if the section is expanded or collapsed.

File: src/molecules/gallery-button/index.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			<Tooltip title="Expand">
+				<IconButton
+					aria-label="show more"
+					onClick={() => {
+						setView("chip");
+					}}
+				>
+					<ViewAgenda />
+				</IconButton>
+			</Tooltip>
```
Comment (pixelass) [inline src/molecules/gallery-button/index.jsx:29]: Most Tooltips have the name "Expand". Keep in mind to change this along the way.

**Review by pixelass — APPROVED** [review]
Looks good to me. 
Take a look at the comments. They can be solved in other pull requests.

File: src/molecules/card-chip-main/index.jsx (hunk 1 above)
Comment (haskamp) [reply src/molecules/card-chip-main/index.jsx:48]: so true, is set to styling ticket to be fixed

File: src/molecules/gallery-button/index.jsx (hunk 2 above)
Comment (haskamp) [reply src/molecules/gallery-button/index.jsx:29]: for got to change that

**Pull request merged**